lua/autorun/server/sv_daily_restart.lua

```diff
@@ -256,7 +269,34 @@ local function tryAlertNotification( secondsUntilNextRestart, msg )
 
         notif:AddButton( "Discard", AlertNotificationDiscardColor )
 
-        notif:Send( player.GetHumans() )
+        notif:Send( noAccess or player.GetHumans() )
+
+        if not msgAdmin then return end
+
+        local notifAdmin = CFCNotifications.new( AlertNotificationAdminName, "Buttons", true )
+
+        notifAdmin:SetTitle( "CFC Daily Restart" )
+        notifAdmin:SetPriority( CFCNotifications.PRIORITY_MAX )
+        notifAdmin:SetDisplayTime( notificationAlertDuration )
+        notifAdmin:SetText( msgAdmin )
+        notifAdmin:SetTextColor( AlertNotificationColor )
+        notifAdmin:SetCloseable( true )
+        notifAdmin:SetIgnoreable( false )
+        notifAdmin:SetTimed( true )
+
+        notifAdmin:AddButton( "Discard", AlertNotificationDiscardColor, false )
+
```

Comment:
could move this and the stuff  above it into a function `newAlertNotif` instead of just repeating it twice